### PR TITLE
Don't return error when adding existing volume

### PR DESCRIPTION
**Pull request by cpuguy83**
Error wasn't really doing anything except for making a bunch of extra
debug logs.

### CODE REVIEW

File: volumes/repository.go [hunk 1]
```diff
@@ -127,7 +127,7 @@ func (r *Repository) get(path string) *Volume {
 
 func (r *Repository) add(volume *Volume) error {
 	if vol := r.get(volume.Path); vol != nil {
-		return fmt.Errorf("Volume exists: %s", volume.ID)
+		return nil
```
Comment (icecrime) [inline volumes/repository.go]: Should the function return `error` if it's always `nil`?

Comment (cpuguy83) [inline volumes/repository.go]: Good point! Will update.

**Comment by cpuguy83** [comment]
Updated.

File: volumes/repository.go [hunk 2]
```diff
@@ -125,12 +124,12 @@ func (r *Repository) get(path string) *Volume {
...
 
-func (r *Repository) add(volume *Volume) error {
+func (r *Repository) add(volume *Volume) {
 	if vol := r.get(volume.Path); vol != nil {
-		return fmt.Errorf("Volume exists: %s", volume.ID)
+		return
 	}
 	r.volumes[volume.Path] = volume
-	return nil
+	return
```
Comment (crosbymichael) [inline volumes/repository.go]: why is this here?

**Comment by cpuguy83** [comment]
@crosbymichael updated

**Comment by crosbymichael** [comment]
LGTM
